Documentation/Signoffs/PLCSubsystem.md

```diff
@@ -0,0 +1,74 @@
+# *Detail Design: PLC Subsystem*
+
+## **Function of the Subsystem:**
+The function of this subsystem is to receive data from the various sensors and relay that information to the HMI display. This system will also send commands to the various subsystems to change behaviors according to the PLC code.
+
+## **Constraints:**
+
+|Segment|Number|Constraint Description|
+|--------|-|-------------------------------------------------|
+|PLC|1|The PLC scan time shall be able to scan fast enough to accommodate the desired sampling rate for all subsystem sensors. These sensors will be sending data every 5 minutes|
+|PLC|2|Communication Bandwidth shall be sufficient enough to handle all data from sensors simultaneously|
+|PLC|3|Maximum Data Latency shall be low enough to allow for data to process on a 5-minute cycle|
+|PLC|4|The PLC shall be required to have an input voltage of 20.0 and 28.0 VDC and a current of at least 140mA to operate within a specific voltage range to ensure compatibility with the power supply system of the overall automation setup. By specifying the input voltage range of 20.0 to 28.0 VDC, we ensure that the PLC can be powered by the C0-01AC without risk of damage or malfunction [4]|
```

Comment:
You are clearly writing the constraint based on the device you have already chosen. That is not a constraint.